### PR TITLE
Add Sass docs (variables, mixins, and loops) to most pages

### PR DESCRIPTION
**Pull request by mdo**
While reviewing #25977, figured I'd take a stab at what we could do for documenting our Sass better. This is similar to our JS docs for methods, options, etc. I think this is doable, just a little more manual than a purely generated mixins docs page.

Preview: <https://deploy-preview-32747--twbs-bootstrap.netlify.app>

TODO:

- [x] Address any comments
- [x] Be consistent with the Sass section place; before or after JS

### CODE REVIEW

**Comment by XhmikosR** [comment]
Looking good so far!

I wonder if it's intentional, button group does not have any Sass section. There might be more components/utilities that are missing.

File: site/content/docs/5.0/forms/validation.md [hunk 1]
```diff
@@ -349,18 +349,34 @@ If your form layout allows it, you can swap the `.{valid|invalid}-feedback` clas
...
+
+Used to iterate over `$form-validation-states` map values to generate our validation styles. Any modifications to the above Sass map will be reflected in your compiled CSS via this loop.
 
 {{< scss-docs name="form-validation-states-loop" file="scss/forms/_validation.scss" >}}
+
+### Customizing
+
+Validation states can be customized via Sass with the `$form-validation-states` map. Located in our `_variables.scss` file, this Sass map is how we generate the default `valid`/`invalid` validation states. Included is a nested map for customizing each state's color, icon, tooltip color, and focus shadow. While no other states are supported by browsers, those using custom styles can easily add more complex form feedback.
+
+Please note that we do not recommend customizing `$form-validation-states` values without also modifying the `form-validation-state` mixin.
```
Comment (XhmikosR) [inline site/content/docs/5.0/forms/validation.md:382]: I wonder if we should make this bold or something?

File: site/content/docs/5.0/utilities/borders.md [hunk 2]
```diff
@@ -77,3 +77,19 @@ Use the scaling classes for larger or smaller rounded corners. Sizes range from
...
+
+{{< scss-docs name="border-radius-variables" file="scss/_variables.scss" >}}
+
+### Mixins
+
+{{< scss-docs name="border-radius-mixins" file="scss/mixins/_border-radius.scss" >}}
+
+### Utilities
+
+TODO: this errored out earlier, unclear why...
```
Comment (XhmikosR) [inline site/content/docs/5.0/utilities/borders.md:95]: Just a comment so that we don't miss this

File: site/content/docs/5.0/forms/overview.md [hunk 3]
```diff
@@ -142,3 +142,15 @@ For situations where it's not possible to include a visible `<label>` or appropr
...
+
+Many form variables are set at a general level to be re-used and extended by individual form components. You'll see these most often as `$btn-input-*` and `$input-*` variables.
+
+### Variables
+
+`$btn-input-*` variables are shared global variables between our buttons and our form components. You'll find these frequently reassigned as values to other component-specific variables.
+
+{{< scss-docs name="input-btn-variables" file="scss/_variables.scss" >}}
+
+### Mixins
```
Comment (XhmikosR) [inline site/content/docs/5.0/forms/overview.md:156]: This is empty

File: site/content/docs/5.0/forms/form-control.md [hunk 4]
```diff
@@ -130,3 +130,21 @@ Learn more about [support for datalist elements](https://caniuse.com/datalist).
...
+
+`$form-label-*` and `$form-text-*` are for our `<label>`s and `.form-text` component.
+
+{{< scss-docs name="form-label-variables" file="scss/_variables.scss" >}}
+
+{{< scss-docs name="form-text-variables" file="scss/_variables.scss" >}}
+
+### Mixins
+
+There are no mixins for form controls.
```
Comment (XhmikosR) [inline site/content/docs/5.0/forms/form-control.md:150]: I wonder if we should just skip any mixins sections if there are no mixins? Seems simpler.

File: site/content/docs/5.0/components/spinners.md [hunk 5]
```diff
@@ -171,6 +171,24 @@ Use spinners within buttons to indicate an action is currently processing or tak
...
+
+{{< scss-docs name="spinner-variables" file="scss/_variables.scss" >}}
+
+### Mixins
+
+There are no mixins for spinners.
+
+### Keyframes
+
+Used for creating the CSS animations for our spinners. Included in the `scss/_spinners.scss` stylesheet.
```
Comment (XhmikosR) [inline site/content/docs/5.0/components/spinners.md:186]: `stylesheet` -> partial or something? The same applies to other places too.

**Review by ffoodd — APPROVED** [review]

**Comment by XhmikosR** [comment]
@mdo this PR contains dis files changes :/

**Comment by mdo** [comment]
Oh woof my bad. I can revert later and fix.

**Comment by XhmikosR** [comment]
@mdo yeah please revert because you committed changes that shouldn't be there :/ Remember to do `npm i` when deps change :)

**Pull request merged**